grid/test-config.ts

```diff
@@ -0,0 +1,5 @@
+import { NockConfig } from '@superfaceai/testing';
+
+export const nockConfig: NockConfig = {
+  testInstance: expect,
```

Comment:
How was this done in previous version? How did we generate hashes for recordings?

Reply:
Mostly in same way (except those tests that did not use `testInstance`).`testInstance` was only moved from `profile`, `provider` and `useCase` parameters to second parameter in `SuperfaceTest` constructor.